db/models/model.js

```diff
@@ -1,7 +1,12 @@
 const db = require("../connection");
+const endPoints = require("../../endpoints.json");
 
 exports.fetchTopics = () => {
   return db.query("SELECT * FROM topics").then(({ rows }) => {
     return rows;
   });
 };
+
+exports.fetchEndpoint = () => {
```

Comment:
You don't really need a model for this, the file can be required in the controller.